pkg/endpoint/endpoint.go

```diff
@@ -82,6 +83,12 @@ var (
 		Requires:    []string{OptionConntrack},
 	}
 
+	OptionSpecConntrackLocal = option.Option{
+		Define:      "CONNTRACK_LOCAL",
+		Description: "Use container local connection tracking table instead of global one",
```

Comment:
I think we are using the word endpoint pretty much everywhere as Cilium may be extended to non container specific use cases. Let's be consistent.